src/tracer/models.ts

```diff
@@ -0,0 +1,22 @@
+import { Event } from '../client';
+import { Evaluation as TestingEvaluation } from '../testing';
+import { BaseEventEvaluator } from '../testing/models';
+import { ArbitraryProperties, PromptTracking } from '../types';
+
+export type TracerEvent = Omit<Event, 'id' | 'traceId'> & { traceId?: string };
```

Comment:
i'd just define `TracerEvent` directly instead of trying to modify `Event` like this, easier to follow